src/app/home/home.js

```diff
@@ -1,8 +1,10 @@
 import {Component} from 'angular2/core';
+import template from './home.html';
 
+const selector = 'sm-home';
 @Component({
-    selector: 'sm-home',
```

Comment:
try to avoid refactoring of non related code
refactor it in separate pull request